server/route/notification.go

```diff
@@ -0,0 +1,19 @@
+package route
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+// CreateNotification craete noitication of project
+func CreateNotification(c *gin.Context) {
+	uuid := c.Param("uuid")
+	c.String(http.StatusOK, "Create notification:"+uuid)
```

Comment:
I expect the response to be as JSON. Will all endpoints response be to be with `Content-Type: application/json`?